### ISSUE TITLE
TinyMCE is not loading in the view

### ISSUE THREAD

**Issue by eduardonwa** [tinymce/tinymce#6657]
Hello there I am developing an application on Laravel 8 and recently I was pulling TinyMCE through a CDN but now I want to install it on the project, but it doesn't appear, what am I doing wrong? I was following these steps:

1. ```php composer.phar require "tinymce/tinymce" ">= 5"```
2.  I placed the script tag like this: ``` <script src="{{asset('js/tinymce.min.js')}}"></script>```

From the network tab I get a 304 code but then a 404 for ```theme.min.js``` are these 2 related then? This is how I boot up:

```
    <script>
        tinymce.init({
        selector: '#body'
        });
    </script>
```

**Comment by JamesToohey** [comment]
Hi @eduardonwa,

Unfortunately we are unable to help with specific integrations of TinyMCE. If you are experiencing an issue within the Editor itself and can reproduce it on a sandbox platform such as [fiddle](https://fiddle.tiny.cloud/) then we are more than happy to help.

My guess is that the TinyMCE assets haven't been configured or deployed correctly and TinyMCE is unable to load them. You could check by making sure `js/themes/silver.min.js` is available via a web browser.

**Comment by eduardonwa** [comment]
Hi @JamesToohey I was able to install TinyMCE because I copied some files from the ```node_modules``` folder over to the ```webpack.mix.js```, now when I want to use it I have to refresh the page (everytime), so it loads but not from the "get-go". Perhaps it's the ```Turbo links``` script, which im not quite sure why "that" is loading, but im sure fixing that will resolve any collateral issues. Thanks!
